src/arch/Lights/LightsDriver_LinuxPacDrive.cpp

```diff
@@ -6,10 +6,8 @@
 #include "LightsDriver_LinuxPacDrive.h"
 
 #include <cstdint>
+#include <../../../extern/libusb/libusb/libusb.h>
```

Comment:
I believe this leads to an unpredictable result. I have libusb in my system, so cmake will use libusb.so (and headers) from system. But header itself is used from git submodules. Different versions of header and compiled so file.
On the system I have to `#include <libusb-1.0/libusb.h>` (if cmake founds it)
-- OR --
 use just libusb/libusb.h or whatever cmake detected and put in include path list